undertow/src/main/java/org/wildfly/swarm/undertow/UndertowFraction.java

```diff
@@ -53,14 +53,14 @@ public void postConstruct() {
 
```

Comment:
Are there changes in this file or is it just formatting?

Maybe update and remove formatting?